website/docs/api/plugins/plugin-content-blog.mdx

```diff
@@ -73,7 +73,7 @@ Accepted fields:
 | `feedOptions.description` | `string` | <code>\`$\{siteConfig.title} Blog\`</code> | Description of the feed. |
 | `feedOptions.copyright` | `string` | `undefined` | Copyright message. |
 | `feedOptions.language` | `string` (See [documentation](http://www.w3.org/TR/REC-html40/struct/dirlang.html#langcodes) for possible values) | `undefined` | Language metadata of the feed. |
-| `sortPosts` | <code>'descending' \| 'ascending' </code> | `'descending'` | Governs the direction of blog post sorting. |
+| `sortPosts` | <code>'descending' \| 'ascending' \| function()</code> | `'descending'` | Governs the direction of blog post sorting. |
```

Comment:
`function` is ambiguous, we should define more explicitly the contract of that function

You should document the TypeScript type here:
https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-blog#types